LLama.SemanticKernel/TextCompletion/LLamaSharpTextCompletion.cs

```diff
@@ -0,0 +1,27 @@
+﻿using LLama;
+using LLama.Abstractions;
+using Microsoft.SemanticKernel.AI.TextCompletion;
+
+namespace Microsoft.SemanticKernel.Connectors.AI.LLama.TextCompletion;
```

Comment:
Should this really be in a `Microsoft.SemanticKernel` namespace instead of something like `LLamaSharp.SenanticKernel`? This seems a bit weird to me, but maybe it's the convention in SemanticKernel.

Reply:
Changed with https://github.com/SciSharp/LLamaSharp/pull/138/commits/2bde188c64cd8b42568aaface7b7cfeb3d4e1648,

I didn't like it either, if ya'll want it back I can change it, but having it be inside this namespace does make more sense and shouldn't affect SemanticKernel itself.